website/client/components/inventory/equipment/equipGearModal.vue

```diff
@@ -151,6 +151,8 @@
       memberOverrideAvatarGear (gear) {
         return {
           [gear.type]: gear.key,
+          hideShield: gear.twoHanded,
```

Comment:
Sorry to give you some more work ... ☹️ 

but I think we should move the two checks `hideShield` and `isEquippedTwoHanded` to `avatar.vue` itself, that way every other (future use) of this component won't have to add this checks if it should hide the shield or the weapon

inside of `avatar.vue` `this.overrideAvatarGear.weapon` would check if the `this.member.items.gear[this.costumeClass][gearType]` is twoHanded and then act the same

Reply:
No problem at all @negue !!

I pushed the latest changes to reflect your comment.

Please let me know if there's anything else you need from me!